src/lib/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -31,10 +31,40 @@ const meta: Meta<typeof MultiSelect> = {
 
 export default meta;
 
-export const Example = {
+export const CondensedExample = {
   args: {
-    items: Array.from({ length: 10 }, (_, i) => ({ label: `Item ${i + 1}`, value: i + 1 })),
-    selectedItems: [{ label: "Item 1", value: 1 }, { label: "Item 2", value: 2 }],
-    disabledItems: [{ label: "Item 1", value: 1 }, { label: "Item 3", value: 3 }],
+    items: Array.from({ length: 10 }, (_, i) => ({
+      label: `Item ${i + 1}`,
+      value: i + 1,
+      selected: true,
```

Comment:
This property only exists here in the story, and doesn't seem to be on the MultiSelectItem type

Reply:
Good spot, missed during cleanup.